### ISSUE TITLE
Latest code

### ISSUE THREAD

**Issue by kenkit** [eret1k/SP-Flash-Tool-src#3]
I tried using the lib files in this project but I can't communicate the with the Download Agent it seems to hang when it starts.

**Comment by kenkit** [comment]
I solved that problem. 
Now if only we could get the latest code

**Comment by rdlab1** [comment]
Can you keep me updated? thank you so much

**Comment by kenkit** [comment]
Yeah, there is flashtool is SPFlash Tool v5.1824 if we could get the updated source it would be good.

**Comment by rdlab1** [comment]
can you compile this version "https://github.com/eret1k/SP-Flash-Tool-src" with your correction?

**Comment by rdlab1** [comment]
Good man!
I tried to look for a newer source, but nothing. I did not understand well sorry, you can fill in the version linked with the modification 0x123456789. Thank you
